src/h5web/visualizations/heatmap/HeatmapVis.tsx

```diff
@@ -1,14 +1,14 @@
 import { Canvas } from 'react-three-fiber';
 import React, { useEffect, useMemo } from 'react';
-import Mesh from './Mesh';
 import { computeTextureData } from './utils';
 import styles from './HeatmapVis.module.css';
 import ColorBar from './ColorBar';
 import ColorMapSelector from './ColorMapSelector';
 import LogScaleToggler from './LogScaleToggler';
 import { useHeatmapState, useHeatmapActions } from './store';
+import { useHeatmapSize as useHeatmapStyles } from './hooks';
```

Comment:
You may want to rename `useHeatmapSize` in `useHeatmapStyles` in the `hooks` file as well